pyprland/plugins/scratchpads/__init__.py

```diff
@@ -760,10 +765,15 @@ async def _handle_focus_tracking(self, scratch: Scratch, active_window: str, act
         if tracker and not tracker.prev_focused_window_wrkspc.startswith("special:"):
             same_workspace = tracker.prev_focused_window_wrkspc == active_workspace
             clients = await self.hyprctl_json("clients")
```

Comment:
I know this was introduced in a former PR, but can you add the "clients" parameter to the hande_focus_tracking function instead of doing a new IPC call?